packages/core/src/config/all.ts

```diff
@@ -126,7 +126,20 @@ const all: PluginStyleguideConfig<'built-in'> = {
     'channels-kebab-case': 'error',
     'no-channel-trailing-slash': 'error',
   },
-  arazzoRules: { spec: 'error', 'parameters-no-body-inside-in': 'off' },
+  arazzoRules: {
+    spec: 'error',
+    'parameters-no-body-inside-in': 'off',
+    'strict-source-description-type': 'error',
+    'arazzo-version-enum': 'off',
```

Comment:
Discussed offline: add `error` to all, recommended (with `warn`), recommended-strict(`error`).

Reply:
Updated, 
@lornajane could you please doublecheck the configuration settings for the rules.

Reply:
Alternatively, maybe we can create a separate ruleset specifically for Spot? Then it would be possible to include those rules by extending from it in *redocly.yaml*, i.e.: 

```yaml
extends:
- recommended
- spot
```